tests/e2e-playwright/tests/platform_CI_tests/test_platform.py

```diff
@@ -85,7 +108,10 @@ def test_simple_workspace_workflow(
         and response.request.method == "POST"
     ) as response_info:
         page.get_by_test_id("newWorkspaceButton").click()
+        page.wait_for_timeout(500)
         page.get_by_test_id("workspaceEditorSave").click()
```

Comment:
so this I don't understand why you add these timeouts. the click already waits 30 seconds by default. what does this bring?

Reply:
In the end I solved it properly like this:
```
        page.get_by_test_id("newWorkspaceButton").click()

        workspace_title_field = page.get_by_test_id("workspaceEditorTitle")
        # wait until the title is automatically filled up
        expect(workspace_title_field).not_to_have_value("")
        page.get_by_test_id("workspaceEditorSave").click()
```
Issue was that playwright was clicking on the save button, but frontend didn't manage yet to change the "dummy" title name. Therefore it got stuck.